hack/gen-pkg-resources.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+cd "$(dirname $0)/.."
+go mod vendor
+
+tmp_dir=$(mktemp -d)
+trap '{ rm -rf -- "$tmp_dir"; }' EXIT
+
+(
+    cd "$tmp_dir"
+    git clone --branch=v1.21.1 --depth 1 https://github.com/kubernetes/cloud-provider-openstack
+    helm template cinder-csi --namespace=kube-system ./cloud-provider-openstack/charts/cinder-csi-plugin > "$tmp_dir/cinder-csi-resources.yaml"
+)
+
+go run ./vendor/github.com/wozniakjan/reverse-kube-resource/main.go \
```

Comment:
```suggestion
go run github.com/wozniakjan/reverse-kube-resource \
```

There is no need for vendor, `go run` can work with modules unless there is something super specific in reverse-kube-resource (which should be fixed in that case).

Reply:
that's a good idea. I can then remove it from `hack/tools.go` at which point it won't pollute the `go.mod`. Thanks!

Reply:
oh no! Please keep it in hack/tools.go, that way we will manage actual version that will be run.

Reply:
ok :)